src/routes/blog/+page.svelte

```diff
@@ -3,10 +3,47 @@
     import { MainFooter, FooterNav, Article } from '$lib/components';
     import { TITLE_SUFFIX } from '$routes/titles.js';
     import { DEFAULT_HOST } from '$lib/utils/metadata';
+    import { _PAGE_NAVIGATION_RANGE } from '$routes/blog/+page';
+    import { page } from '$app/stores';
+    import { onMount } from 'svelte';
 
     export let data;
+    const featured = data.featured;
 
-    const featured = data.posts.find((post) => post.featured);
+    let currentPage = 1;
+
+    $: currentPageRange = (): number[] => {
+        const chunkIndex = Math.floor((currentPage - 1) / _PAGE_NAVIGATION_RANGE);
```

Comment:
We are calculating the "chunckIndex" or "batchIndex" 3 different times. This can be it's own reactive statement probably :+1:

Reply:
If we remove the "chunkIndex" then this can be a single line too